CONTRIBUTING.md

```diff
@@ -172,19 +173,38 @@ Describe a new checker in [checkers section](./README.md#checkers).
 ### error-compare
 
 ```go
-❌   assert.ErrorContains(t, err, "not found")
-     assert.EqualError(t, err, "user not found")
+❌   assert.Contains(t, err.Error(), "not found")
      assert.Equal(t, err.Error(), "user not found")
      assert.Equal(t, err, errSentinel) // Through `reflect.DeepEqual` causes error strings to be compared.
      assert.NotEqual(t, err, errSentinel)
      // etc.
 
+✅   assert.ErrorContains(t, err, "not found")
+     assert.EqualError(t, err, "user not found")
```

Comment:
I'm a bit afraid this one is valid

Reply:
My point is that ErrorEqual should be part of the sane checker than ErrorContains.

It should be discouraged

Reply:
Maybe, let's wait for Anton's feedback

Reply:
@Antonboom maybe you missed this old thread 

https://github.com/Antonboom/testifylint/pull/162/commits/4cb531e2e45e911b6ea77f3616b8c0e41dda42dd#r1661489366

You said ErrorContains is an antipattern, what is your point of view about ErrorEqual ?